### ISSUE TITLE
Redis functionality very limited + doesn't find all cache keys

### ISSUE THREAD

**Issue by amitai52** [TablePlus/TablePlus#1552]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Working with Redis, vs AWS server version 5.0.4 over SSH tunnel I open outside TablePlus

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
2.11 build 276

3. There are a few problems:
a. We use several namespaces like:
resque:subnamespace1:*
and
cache:subnamespace2:*

The client doesn't read all the namespaces, and doesn't see all the keys.
We have 120,000 keys in our DB, most in the 2nd namespace, all are ignored.
TP reads only the first namespace with about 36 keys.

b. reading is very slow ... it take a long time to read ~36 keys . ...
you can see them loading one by one about 1 second per 2-3 keys.
I can't imagine how 120k keys would load...

c. I prefer to see namespaces as folders you can open / drill down into
and then look at keys just under than namespace / sub-namespace.
this is more useful, and also more efficient in getting data

**Comment by maxgalbu** [comment]
yep seems like it doesn't do any subsequent scan, and pagination doesn't work
